src/components/IncidentSeverityTag/index.tsx

```diff
@@ -5,11 +5,18 @@ import {
 } from "react-icons/hi";
 import { IncidentSeverity } from "../../api/services/incident";
 
-const getSeverity = (severity: IncidentSeverity) => {
+const getSeverity = (severity: IncidentSeverity | string) => {
   switch (severity) {
-    case 2:
+    case "4":
```

Comment:
I don't understand why we have these numbers ?

Reply:
incident severity saved in the db is a number. we have this number to severity mapping because of that.

Reply:
Okay, I'll make changes from frontend and we need to write a db script or have work around in the front end to support both values.